### PR TITLE
RA-1827:  Collapsible "if" statements should be merged

### PR DESCRIPTION
**Pull request by DBryzz**
[RA-1827](https://issues.openmrs.org/browse/RA-1827):  Collapsible "if" statements should be merged.

Collapsible "if" statements, as described [here](https://issues.openmrs.org/browse/RA-1827), has been merged.

Please review and merge if all is ok.

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientdashboard/VisitIncludesFragmentController.java [hunk 1]
```diff
@@ -39,9 +39,8 @@ public void controller(FragmentConfiguration config,
 		Object patient = config.get("patient");
 		Object visitType = config.get("visitType");
 
-		if(patient == null) {
+		if( (patient == null) && (request.getParameter("patientId") != null) ) {
```
Comment (sacull) [inline omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientdashboard/VisitIncludesFragmentController.java:42]: image
You have made unnecessary spaces in the places marked in red. Additionally, it is good style to make a space between the "if" and the opening brace.

**Review by sacull — CHANGES_REQUESTED** [review]
image
It is good to include the ticket signature from JIRA in commit messages.

Additionally, in the PR description, it is worth adding a link to the JIRA ticket.

File: omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientdashboard/VisitIncludesFragmentController.java (hunk 1 above)
Comment (DBryzz) [reply omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientdashboard/VisitIncludesFragmentController.java]: Thanks for the correction [Lukasz Debicki](https://github.com/sacull)
I have effected the changes

File: omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientdashboard/VisitIncludesFragmentController.java [hunk 2]
```diff
@@ -50,7 +49,6 @@ public void controller(FragmentConfiguration config,
 						patient = patientService.getPatient(NumberUtils.toInt(patientId));
 					}
 				}
-			}
```
Comment (sacull) [inline omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientdashboard/VisitIncludesFragmentController.java:53]: image
Don't you have one tab too many in the marked column?

**Review by sacull — CHANGES_REQUESTED** [review]

File: omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientdashboard/VisitIncludesFragmentController.java (hunk 2 above)
Comment (sacull) [reply omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientdashboard/VisitIncludesFragmentController.java:53]: You removed one if nesting, so the entire block should move one tab to the left.

Comment (DBryzz) [reply omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientdashboard/VisitIncludesFragmentController.java:53]: new changes made

**Review by sacull — APPROVED** [review]
Now it looks good to me.
@mogoodrich @dkayiwa

**Comment by dkayiwa** [comment]
@sacull no need to tag. Just approve and comment with looks good to you. Then we shall do the needful.
Otherwise, thank you so much for the great reviews that you are doing! 👍

**Comment by sacull** [comment]
@dkayiwa I understand. I don't know how you see this PR, whether you get any notifications or not, that's why I was calling people who can merge. Now I know I don't have to do this. Thanks. ;)

**Comment by dkayiwa** [comment]
We get notifications for every pull request comment. 😊

**Pull request merged**